frontend/src/components/OdhExploreCard.tsx

```diff
@@ -14,7 +14,11 @@ type OdhExploreCardProps = {
 const OdhExploreCard: React.FC<OdhExploreCardProps> = ({ odhApp, isSelected, onSelect }) => {
   const cardClasses = classNames('odh-card', { 'm-disabled': odhApp.spec.comingSoon });
   const badgeClasses = classNames('odh-card__partner-badge', {
-    'm-warning': odhApp.spec.support === 'other',
+    'm-warning': odhApp.spec.category === 'Third party support',
+    'm-hidden': odhApp.spec.category === 'Red Hat',
+  });
+  const supportedImageClasses = classNames('odh-card__supported-image', {
+    'm-hidden': odhApp.spec.category !== 'Red Hat',
```

Comment:
review/optional: line 18 and 21 look similar to the comments about a check for `odhApp.spec.category` being "defined", is that concern here too?

Reply:
ditto 😉